docs/eft-log-reference/EFT_LOG_EVENT_DICTIONARY.md

```
@@ -0,0 +1,368 @@
+| Popup from server                                       | `NotificationPopup` (message/image, button flags/duration)                                                                                                                   | —                                                       |
+
+### Economy / items
+
+| Goal                                          | Primary signal                                                                                                                                         | Cautions                                      |
+| --------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------- |
+| Flea sale                                     | `push-notifications`: `RagfairOfferSold` (sold item, count, buyer nickname, handbookId)                                                                | Buyer nickname is PII                         |
+| Flea rating change                            | `RagfairNewRating` (`isRatingGrowing`, rating)                                                                                                         | —                                             |
+| Balance / purchase / stash rows / battle-pass | `ExpansionsAccountBalanceIncreased`, `ExpansionsAccountOfferPurchased`, `ExpansionsAccountTarcoinBalance`, `StashRows`, `BattlePassUniversalDocuments` | Amounts are account data                      |
+| Insurance cost                                | `backend` `/client/insurance/items/list/cost` (E and P)                                                                                                | Only cost route exists                        |
+| Insurance reconciliation problem              | `insurance`: `Items to insure does not contain: <ITEM>` (C: 5 builds)                                                                                  | No success/payment/expiry/return events exist |
+| Inventory rejected                            | `inventory`: `[<ID>                                                                                                                                    | <NICK>                                        | Profile]<CORR> - Client operation rejected by server:<CODE> - OperationType:<OP>` | Failure-only evidence; no universal commit marker |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)